pkg/ddevapp/config.go

```diff
@@ -1059,6 +1059,12 @@ ENV N_INSTALL_VERSION="%s"
 	if app.CorepackEnable {
 		extraWebContent = extraWebContent + "\nRUN corepack enable"
 	}
+	if app.Type == nodeps.AppTypeDrupal11 {
```

Comment:
Is this pinning only for Drupal 11? Is pinning for Drupal 11 what we should actually be doing instead of the current approach?

Reply:
> Is this pinning only for Drupal 11?
I pin it to 3.40.1 when it's not Drupal 11, so it will be downgraded if people do a full upgrade or install something that requires sqlite3=3.40.1, and unpin it here by renaming the file (so it won't be downgraded for Drupal 11).

> Is pinning for Drupal 11 what we should actually be doing instead of the current approach?
We cannot pin it to 3.45.1 because there is no 3.45.1 in the repo.